apps/backend/src/modules/groups/user-groups-to-permissions.model.ts

```diff
@@ -0,0 +1,20 @@
+import {
+	AbstractModel,
+	DatabaseTableName,
+} from "~/libs/modules/database/database.js";
+
+class UserGroupsToPermissionsModel extends AbstractModel {
+	public permissionId!: number;
+
+	public userGroupId!: number;
+
+	public static override get idColumn(): string {
+		return "id";
```

Comment:
I can't get why you return "id" string here, could you explain?

Reply:
Maybe it's not needed. I just thought of adding this to tables that represent a many-to-many relationship.